pkgs/tools/networking/ddclient/default.nix

```diff
@@ -5,7 +5,7 @@ perlPackages.buildPerlPackage rec {
   version = "3.9.1";
 
   src = fetchurl {
-    url = "mirror://sourceforge/ddclient/${pname}-${version}.tar.gz";
+    url = "https://github.com/ddclient/${pname}/archive/refs/tags/v${version}.tar.gz";
```

Comment:
This should probably be using `fetchFromGitHub`

Reply:
Yes, it should.